<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>779</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>455</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SEL-735 PQ Reporting</string>
  </property>
  <widget class="QLabel" name="scheduleLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Schedule</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="nrs048CheckBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NRS048 Standard Report</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="customReportLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>450</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Create A Custom Report:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previewReportPushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>670</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview Report</string>
   </property>
  </widget>
  <widget class="QLabel" name="screenSizeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>730</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Connection Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="connectionStatusLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>730</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="dailyCheckBox">
      <property name="text">
       <string>Daily</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="weeklyCheckBox">
      <property name="text">
       <string>Weekly</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="monthlyCheckBox">
      <property name="text">
       <string>Monthly</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>921</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="meterListLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Meter List</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="reportConfigLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Report Configuration</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>490</y>
     <width>221</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="vssiCheckBox">
      <property name="text">
       <string>Voltage Sag, Swell, Interruptions</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="harmonicsCheckBox">
      <property name="text">
       <string>Harmonics (3,5,7)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ldpCheckBox">
      <property name="text">
       <string>Load Profile Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="eventsCheckBox">
      <property name="text">
       <string>PQ Events</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="reportConfigLabel_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>456</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TEAM Software Host Details</string>
   </property>
  </widget>
  <widget class="QLabel" name="hostNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Host Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="portNumberLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>API Port Number:</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="saveCommandLinkButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>720</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>191</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="hostNameLineEdit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>localhost</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="portNumberLineEdit">
      <property name="placeholderText">
       <string>5630</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="testConnectionpushButton">
      <property name="text">
       <string>Test Connection</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="reportsFolderLineEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>630</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="reportsFolderLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>610</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reports Folder Path</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="meterTableWidget">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>191</y>
     <width>341</width>
     <height>461</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>676</y>
     <width>341</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="updateListPushButton">
      <property name="text">
       <string>Update List</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveSelectedMeterListPushButton">
      <property name="text">
       <string>Save Selection</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="fileDialogPushButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>630</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
